include/sdbus-c++/ProxyInterfaces.h

```diff
@@ -141,6 +141,14 @@ namespace sdbus {
         {
         }
 
+        /*!
+         * @brief return object path of the underlying DBus object
+         */
+        const std::string& getObjectPath() const
+        {
+            return getProxy().getObjectPath();
+        }
```

Comment:
Ditto, moving to the bottom of the class...

Reply:
Done